stdlib/public/Platform/Platform.swift

```diff
@@ -103,8 +103,7 @@ public var errno : Int32 {
   get {
 #if os(OSX) || os(iOS) || os(watchOS) || os(tvOS) || os(FreeBSD)
     return __error().pointee
-//FIXME: os(Windows) should be replaced, such as triple(Cygwin)
```

Comment:
@jakepetroules, You have a point there. Is it will be OK if we introduce the new ```env()``` and that exactly defined by the ```EnvironmentType``` of the [LLVM tripple](http://llvm.org/docs/doxygen/html/classllvm_1_1Triple.html)?  Or, do you have any other name and definition to suggest?

I think we need more opinions. (CC: @jckarter, @rjmccall)